tests/Unit/HC/abort.cpp

```diff
@@ -0,0 +1,93 @@
+
+// RUN: %hc %s -o %t.out && %t.out
```

Comment:
@gargrahul the test is incorrect as @scchan pointed out. Once you create a test you should run it with `make test` to make sure it works.